mllib/src/main/scala/org/apache/spark/ml/regression/GBTRegressor.scala

```diff
@@ -134,11 +146,15 @@ class GBTRegressor @Since("1.4.0") (@Since("1.4.0") override val uid: String)
 
 @Since("1.4.0")
 object GBTRegressor extends DefaultParamsReadable[GBTRegressor] {
-
-  /** Accessor for supported loss settings: squared (L2), absolute (L1) */
+  /** Accessor for supported loss settings: squared (L2), absolute (L1), gaussian (squared),
```

Comment:
style: follow other multi-line docstrings.  Also, make it clearer that gaussian and laplace are aliases for squared and absolute.